clients/src/main/java/org/apache/kafka/common/requests/JoinGroupRequest.java

```diff
@@ -70,6 +70,21 @@ public static void validateGroupInstanceId(String id) {
         });
     }
 
+    /**
+     * Ensures that the provided {@code reason} remains within a range of 255 chars.
+     * @param reason This is the reason that is sent to the broker over the wire
+     *               as a part of {@code JoinGroupRequest}, {@code LeaveGroupRequest}
+     *               or {@code RemoveMembersFromConsumerGroupOptions} messages.
```

Comment:
nit: We can remove this line because, in the end, we also send a LeaveGroupRequest in this case.

Reply:
though it looks like a straightforward change, probably I need to spend more time digesting it.